Cargo.toml

```diff
@@ -110,7 +110,7 @@ serde_json = "1.0"
 serial_test = "3.1"
 syn = "2.0"
 tempfile = "3.13"
-testcontainers = "0.23"
+testcontainers = "0.23.1"
```

Comment:
do we want patch versions? @pablodeymo

Reply:
you are right, better to include it without patch number